README.rst

```diff
@@ -106,22 +106,36 @@ Example:
 
    $ mkdir test && cd test && git init
```

Comment:
```suggestion
```
Don't really need this, as it is a given that we will have  python files and be in a git directory. instead consider an example showing an error thrown when we are not in a git directory, with a comment re-statating that you need to be in a git directory. Those are often easier to find when people scan the docs while troubleshooting

Reply:
What I intended to do in this example is give the reader a complete set of command lines they can copy-paste on the terminal, and have the exact thing described happen on their own disk.

But you have a good point on troubleshooting. I'll try adding that.